llumnix/backends/vllm/scheduler.py

```diff
@@ -205,6 +206,25 @@ def schedule(self) -> Tuple[List[SequenceGroupMetadata], SchedulerOutputs]:
         self.update_instance_info_callback(self._get_instance_info())
         return seq_group_metadata_list, scheduler_outputs
 
+    def _schedule_running(self, *args, **kwargs):
+        args_list = list(args)
```

Comment:
Using `def _schedule_running(self, runging_queue:deque , *args, **kwargs):`, args_list[0] is confusing.